lib/typesense.ts

```diff
@@ -0,0 +1,27 @@
+import TypesenseInstantSearchAdapter from "typesense-instantsearch-adapter";
+
+export function typesenseSearch({
+  queryBy,
+  defaultSort,
+}: {
+  queryBy: string[];
+  defaultSort?: string;
+}) {
+  const searchAdapter = new TypesenseInstantSearchAdapter({
+    server: {
+      apiKey: "FByczfHEjsTCihgkkYlq2YbAgUKMoyVP", // Be sure to use the search-only-api-key
```

Comment:
I think we should start storing these secrets on an environment variable. @zainfathoni what do you think?

Reply:
Agreed. Just let me or @resir014 know what environment variable to store, Mbak @ekamuktia, we can do it for you. 😁